third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-if [[ ! -v K32W061_SDK_ROOT ]]; then
```

Comment:
This seems to not be a documentation change - should we call it out in the description?

Reply:
It is a check to make it compatible for linux as well as macos. The line checks if the k32w sdk exists before applying the patch. This has been verified to work.

Reply:
No, I don't get it.  Shouldn't this be `-d $K32W061_SDK_ROOT`?  

What, exactly, has been verified to work?  I just tested this locally, and I get:
```
% mkdir /tmp/foo                                      
% env K32W061_SDK_ROOT=/tmp/foo third_party/k32w_sdk/mr2_fixes/patch_k32w_mr2_sdk.sh
K32W061_SDK_ROOT is not set
```

Reply:
If you refer to https://github.com/project-chip/connectedhomeip/blob/master/examples/lock-app/k32w/README.md
the document explains how to download the SDK, Once the SDK is downloaded, "export K32W061_SDK_ROOT=path_to_SDK" has to be executed as explained in the documentation.

Reply:
I understand all that, but that does not affect whether this specific line works, and this specific line does not work.  Can you please paste in a shell session showing you running these comands against the modified script?

Reply:
Ok I get it.. Thanks!! have made an update